lib/dor/services/client/object_version.rb

```diff
@@ -6,6 +6,23 @@ class Client
       # API calls that are about versions
       class ObjectVersion < VersionedService
         Version = Struct.new(:versionId, :tag, :message, keyword_init: true)
+        VersionStatus = Struct.new(:versionId, :open, :openable, :assembling, :accessioning, :closeable, keyword_init: true) do
+          def openable?
+            openable
+          end
```

Comment:
Should `open` also have a question-mark method?

Reply:
Yes. Done.